lib/Conversion/TritonGPUToLLVM/GCNAsmFormat.cpp

```diff
@@ -125,6 +125,17 @@ GCNInstr::Operand *GCNBuilder::newAddrOperand(mlir::Value addr,
   return opr;
 }
 
+GCNInstr::Operand *GCNBuilder::newEmptyOperand(std::string arg) {
```

Comment:
I would suggest the same naming as in `PTXBuilder`:
`Operand *newConstantOperand(const std::string &v)`

Also it doesn't need `std::stringstream` - is it possible in our case too?

Reply:
> Also it doesn't need std::stringstream - is it possible in our case too?

Yes, I just copied this code from earlier commit without really checking the code...